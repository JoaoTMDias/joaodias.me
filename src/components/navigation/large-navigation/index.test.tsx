import React from "react";
import { render, cleanup } from "@testing-library/react";

import { LargeNavigation } from "./index";

afterEach(cleanup);

describe("<LargeNavigation />", () => {
	it("should render without errors", () => {
		const component = render(<LargeNavigation />);

		expect(component).toMatchSnapshot();
	});
});
