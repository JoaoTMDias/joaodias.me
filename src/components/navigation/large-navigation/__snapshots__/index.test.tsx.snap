// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LargeNavigation /> should render without errors 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="sc-AxjAm fnTFax"
        id="top-navigation"
        title="Top Navigation"
      >
        <ul
          class="sc-AxirZ dQyERV"
        >
          <li
            class="sc-AxiKw bmyGDD"
          >
            <a
              aria-label="Go to the about page"
              class="large-nav__link"
              href="/about/"
              title="Go to the about page"
            >
              About
            </a>
          </li>
          <li
            class="sc-AxiKw bmyGDD"
          >
            <a
              aria-label="Go to the contacts page"
              class="large-nav__link"
              href="/contact/"
              title="Go to the contacts page"
            >
              Contact
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="sc-AxjAm fnTFax"
      id="top-navigation"
      title="Top Navigation"
    >
      <ul
        class="sc-AxirZ dQyERV"
      >
        <li
          class="sc-AxiKw bmyGDD"
        >
          <a
            aria-label="Go to the about page"
            class="large-nav__link"
            href="/about/"
            title="Go to the about page"
          >
            About
          </a>
        </li>
        <li
          class="sc-AxiKw bmyGDD"
        >
          <a
            aria-label="Go to the contacts page"
            class="large-nav__link"
            href="/contact/"
            title="Go to the contacts page"
          >
            Contact
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
